config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits-master.yaml

```diff
@@ -42,3 +48,29 @@ presubmits:
       - image: golang:1.20
         command:
         - scripts/test-infra/build.sh
+  - name: pull-node-feature-discovery-verify-e2e-test-master
+    branches:
+    - ^master$
+    max_concurrency: 1
+    decorate: true
+    always_run: false
+    annotations:
+      testgrid-dashboards: sig-node-node-feature-discovery
+      testgrid-tab-name: e2e-test-presubmit
+      description: "Run end-to-end tests for node-feature-discovery before merging a pull request"
```

Comment:
Could you change the description: 
"Verify regresions by running e2e-test" 
or something similar

Reply:
done